Client/Frontend/UserContent/UserScripts/FullscreenHelper.js

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+/*var $<possiblySupportedFunctions> = {
+    request: ["requestFullscreen", "webkitRequestFullscreen", "webkitRequestFullScreen", "mozRequestFullScreen", "msRequestFullscreen"],
+    exit: ["exitFullscreen", "webkitCancelFullScreen", "webkitExitFullscreen", "mozCancelFullScreen", "msExitFullscreen"],
+    enabled: ["fullscreenEnabled", "webkitFullscreenEnabled", "mozFullScreenEnabled", "msFullscreenEnabled"],
+    element: ["fullscreenElement", "webkitFullscreenElement", "webkitCurrentFullScreenElement", "mozFullScreenElement", "msFullscreenElement"],
+    change: ["fullscreenchange", "webkitfullscreenchange", "mozfullscreenchange", "MSFullscreenChange"],
+    error: ["fullscreenerror", "webkitfullscreenerror", "mozfullscreenerror", "MSFullscreenError"]
+}*/
+
+var $<isFullscreenSupportedNatively> = document.fullscreenEnabled ||
```

Comment:
`let` or `const` if unedited probably.

Reply:
For the first comment, yes it's passive.
For the second comment, I think it would affect BOTH but atm, I don't see a difference. IE: I don't see it causing any problems.

I tested in both Mobile UA and Desktop UA. Alternatively, I can just make it inject ONLY if the user agent is Desktop. I think that might be best anyway so I'll update the PR to do that.